examples/bri-3/prisma/migrations/20221012172320_add_bpi_subject_account_table/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE "BpiSubjectAccount" (
+    "id" TEXT NOT NULL,
+    "creatorBpiSubjectId" TEXT NOT NULL,
+    "ownerBpiSubjectId" TEXT NOT NULL,
+
+    CONSTRAINT "BpiSubjectAccount_pkey" PRIMARY KEY ("id")
+);
+
+-- AddForeignKey
+ALTER TABLE "BpiSubjectAccount" ADD CONSTRAINT "BpiSubjectAccount_creatorBpiSubjectId_fkey" FOREIGN KEY ("creatorBpiSubjectId") REFERENCES "BpiSubject"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
isn't it possible to add the Foreign key constraint during table creation, like we do with plain SQL.

Reply:
this is auto generated by prisma, i guess because i had to manually name fk constraints because there are 2 referencing same table